front/src/modules/search/components/CommandMenu.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import {
+  StyledDialog,
+  StyledEmpty,
+  StyledGroup,
+  StyledInput,
+  StyledItem,
+  StyledList,
+  // StyledSeparator,
+} from './CommandMenuStyles';
+
+export const CommandMenu = ({ initiallyOpen = false }) => {
+  const [open, setOpen] = React.useState(initiallyOpen);
+
+  // Toggle the menu when ⌘K is pressed
+  React.useEffect(() => {
```

Comment:
We have a hook for this : 

`import { useHotkeys } from 'react-hotkeys-hook';`

Reply:
👍 thanks